ui/src/common/layouts/CollectionsMenu/CollectionsMenu.scss

```diff
@@ -6,3 +6,37 @@
     display: none;
   }
 }
+
+.ant-dropdown-menu {
```

Comment:
I know that it's ugly but there was no way to override antd styling (if we don't want to use submenu).

Reply:
that's not only ugly but also affect all `ant-dropdown-menu`, since whatever you leave out of `.__CollectionsMenu__` is a global style.

Whatever is rendered under `CollectionsMenu`, should have their style in `__CollectionsMenu__` class, such as the `link` class.

Since dropdown is rendered outside of the `CollectionsMenu` DOM-wise. Only way to scope the style is to use `overlayClassName` of `Dropdown` that is rendered by `DropdownMenu`


```jsx
<DropdownMenu overlayClassName="more-collections-menu"...
```

```scss
.more-collections-menu {
  // styles, classes
}
```